examples/cdk/lib/example-function.MyFunctionWithDecorator.ts

```diff
@@ -36,28 +38,34 @@ export class MyFunctionWithDecorator {
     metricWithItsOwnDimensions.addMetric('single-metric', MetricUnits.Percent, 50);
 
     // ### Experiment tracer
-    tracer.putAnnotation('Myannotation', 'My annotation\'s value');
 
-    // Create subsegment & set it as active
-    const segment = tracer.getSegment(); // This is the facade segment (the one that is created by Lambda & that can't be manipulated)
-    const subsegment = segment.addNewSubsegment('MySubSegment');
+    // Service & Cold Start annotations will be added for you by the decorator/middleware
 
+    // These traces will be added to the main segment (## index.handler)
+    tracer.putAnnotation('awsRequestId', context.awsRequestId);
+    tracer.putMetadata('eventPayload', event);
+
+    // Create another subsegment & set it as active
+    const handlerSegment = tracer.getSegment(); // This is the custom segment created by Tracer for you (## index.handler)
+    const subsegment = handlerSegment.addNewSubsegment('### MySubSegment');
     tracer.setSegment(subsegment);
-    // TODO: Add the ColdStart annotation !!! NOT POSSIBLE
-    // tracer.putAnnotation('ColdStart', tracer);
 
+    let res;
     try {
-      throw new Error('test');
-      // Add the response as metadata
+      res = { foo: 'bar' };
     } catch (err) {
```

Comment:
Same as above. Why try/catch without throw?

Reply:
See [this comment](https://github.com/awslabs/aws-lambda-powertools-typescript/pull/439#discussion_r780100117).

Reply:
This one has the throw at [L57](https://github.com/awslabs/aws-lambda-powertools-typescript/blob/4ac474d6a5e76f00bff4c5d3c597d757c724b113/examples/cdk/lib/example-function.MyFunctionWithDecorator.ts#L57).